### ISSUE TITLE
Feature request: navigation between issue articles when viewing an article

### ISSUE THREAD

**Issue by deevroman** [TheFive/osmbc#1019]
In this request I will try to write how my typical work on a release is done and why I need this feature.

---

I usually translate each article in a separate tab.

It's more convenient because the page doesn't reload constantly when I lose focus.

For example, when I want to copy something from the original

https://github.com/TheFive/osmbc/assets/25708359/706cb6a7-d263-4322-bf9c-0a2764339508

Or when I want to click on a link in an article. Or when I want to translate a piece of text with [another translator](https://addons.mozilla.org/ru/firefox/addon/traduzir-paginas-web)

https://github.com/TheFive/osmbc/assets/25708359/657d862d-891e-482e-9e33-5d8e2cd95158

This makes it easier for me to open each article separately

<img width="1509" alt="Снимок экрана 2024-01-05 в 01 22 49" src="https://github.com/TheFive/osmbc/assets/25708359/ac632b37-ba4f-4df1-bdd8-330589c9eb5b">

But this method has a disadvantage: you have to go back to the list of articles

## Alternatives

Open all articles at once, for example, using [this extension](https://addons.mozilla.org/ru/firefox/addon/snaplinksplus/)

I apologize for the small DOS 🙈

https://github.com/TheFive/osmbc/assets/25708359/216e9e68-a068-4f98-922d-0aaeb36c25cc

I only recently found out about this extension, and I can't tell yet if it's convenient.

---

If I don't know any convenient ways to translate, please share.

**Comment by TheFive** [comment]
Yes this translation is getting more and more and not all basic of OSMBC are supporting that.

so in the blog view you would like to have BOTH languages in edit mode, to see the markdown of the original and the translated article on ONE page, this can be a change that is  of interest of all editors.


as an alternativ you suggest just a "foreward backward" button in an article to move in one blog to the next or the previous article. 

Correct ?

**Comment by deevroman** [comment]
> BOTH languages in edit mode

Hmm, maybe sometimes that makes sense. But it won't save you from constantly reloading (and closing the text field) when you lose focus. For example, when you need to switch between the news source and OSMBC.
